com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -278,6 +278,8 @@ public static bool DrawHeaderFoldout(string title, bool state, bool isBoxed = fa
         /// <param name="isBoxed"> [optional] is the eader contained in a box style ? </param>
         /// <param name="hasMoreOptions"> [optional] Delegate used to draw the right state of the advanced button. If null, no button drawn. </param>
         /// <param name="toggleMoreOptions"> [optional] Callback call when advanced button clicked. Should be used to toggle its state. </param>
+        /// <param name="documentationURL">[optional] A url that will be open when the help button on the header is pressed</param>
+        /// <param name="contextAction">[optional] The callback executed when the burger menu on the header is pressed</param>
```

Comment:
The callback that the Unity Editor executes when the user presses the burger menu on the header.